src/Element.elm

```diff
@@ -2,6 +2,8 @@ module Element exposing (..)
 
 import Html exposing (Attribute, Html)
 import Html.Attributes exposing (style)
+import Debug exposing (toString)
```

Comment:
`toString` used to be a general thing, relying on javascript's `anything.toString()`, but now, it's only used for debugging. if you use it, you can't compile a production build!

in this case, use `String.fromInt` instead to go `Int -> String` like we need